hmrc-mongo-play-26/src/main/scala/uk/gov/hmrc/mongo/play/json/Codecs.scala

```diff
@@ -35,7 +36,18 @@ trait BsonConversion {
 object BsonConversion extends BsonConversion
 
 trait Codecs {
-  def playFormatCodec[A](format: Format[A])(implicit ct: ClassTag[A]): Codec[A] = new Codec[A] {
+  val logger = LoggerFactory.getLogger(classOf[Codecs].getName)
+
+  /** @param legacyNumbers `true` will preserve the Number modifications which occured with simple-reactivemongo when storing
+    * extremely large and small numbers.
+    * The default value `false` should suffice since the affected numbers are not within typical usage (e.g. 4.648216657858037E+74),
+    * and if occur, should probably  be stored and retrieved accurately.
```

Comment:
Tthe last line is a little confusing I think. Maybe something like:

'The default value `false` should be preferred in most cases. This does change the previous behaviour from reactivemongo, but only for extreme values not within typical usage (e.g. 4.648216657858037E+74). This ensures that should numbers in this extreme range occur, they will be stored and retrieved accurately, whereas with the legacy behaviour they may be modified in unexpected ways.